rkt/api_service.go

```diff
@@ -38,11 +38,14 @@ import (
 	"github.com/coreos/rkt/store"
 	"github.com/coreos/rkt/version"
 	"github.com/godbus/dbus"
+	"github.com/hashicorp/golang-lru"
 	"github.com/spf13/cobra"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 )
 
+const defaultCacheSize = 128
```

Comment:
I see this size is unitless, can we get a comment expliciting that this is the number of entries in the LRU cache?